### PR TITLE
Update decoupled CMS guide for RC2 (Lombiq Technologies: OCORE-32)

### PR DESCRIPTION
**Pull request by Piedone**
Note that apart from the bare minimum, as in removing pre-RC2 references I also:
- Updated the alias-pattern.jpg screenshot that shows the new default pattern of Alias Part. However, this is now bit different than all the other screenshots since I'm using a different browser (the original Edge is not a current browser anymore).
- Added some clarification about using the guide.
- Fixed typos/grammar.

### CODE REVIEW

File: src/docs/guides/decoupled-cms/README.md [hunk 1]
```diff
@@ -63,17 +66,13 @@ The newly created website should be able to run, and look like this:
 
 This will allow for the Razor Pages to be reloaded without the need to recompile them.
 
-- Modify the `<ItemGroup>` section like this:
+- Add a new `<ItemGroup>` section like this:
 
 ```xml
 <ItemGroup>
-  <PackageReference Include="OrchardCore.Application.Cms.Targets" Version="1.0.0-rc1-10004" />
+  <PackageReference Include="OrchardCore.Application.Cms.Targets" Version="1.0.0-rc2-13450" />
```
Comment (hishamco) [inline src/docs/guides/decoupled-cms/README.md:73]: Can we made the version `1.0.*` to avoid change it until `1.1.*`

**Comment by deanmarcussen** [comment]
LGTM, good to remove the references to nightly feeds etc as they are no longer necessary.

Screenshots are annoying, but one day we will have @jptissot Cypress pr, and might be able to automate screen shot taking during the tests :)

File: src/docs/guides/decoupled-cms/README.md (hunk 1 above)
Comment (Piedone) [reply src/docs/guides/decoupled-cms/README.md:73]: That would move the actually used version though if the user uses the preview feed. I think that referencing a particular version in this guide is better even if we need to update it with new releases.

**Comment by sebastienros** [comment]
Please change the package to Core instead of CMS. We had to do it sure to a bug in rc1

File: src/docs/guides/decoupled-cms/README.md (hunk 1 above)
Comment (Piedone) [reply src/docs/guides/decoupled-cms/README.md:73]: @sebastienros you mean this should reference `OrchardCore.Application.Cms.Core.Targets` instead? Yeah, I see that.

Comment (sebastienros) [reply src/docs/guides/decoupled-cms/README.md:73]: Yes this one, thanks

Comment (Piedone) [reply src/docs/guides/decoupled-cms/README.md]: Changed it.

**Pull request merged**